src/studio/src/designer/backend/Infrastructure/GitRepository/AltinnGitRepository.cs

```diff
@@ -99,6 +101,16 @@ public IList<AltinnCoreFile> GetSchemaFiles()
             return altinnCoreSchemaFiles;
         }
 
+        /// <summary>
+        /// Finds all text files regardless of location in repository.
+        /// </summary>
+        public List<string> GetLanguageFiles()
```

Comment:
We should change this to return IList<AltinnCoreFile> to be consistent with the GetSchemaFiles method above.